### PR TITLE
Use check_var for WORKAROUND_DEPS and BREAK_DEPS in deal_with_dependency_issues

### PR DESCRIPTION
**Pull request by hjluo**
We need use check_var for WORKAROUND_DEPS BREAK_DEPS in deal_with_dependency_issues

- Related ticket: https://progress.opensuse.org/issues/54668
- Needles: N/A
- Verification run: http://openqa.suse.de/tests/3249847

### CODE REVIEW

File: lib/y2_installbase.pm [hunk 1]
```diff
@@ -98,10 +98,10 @@ sub deal_with_dependency_issues {
...
         send_key 'ret';
     }
-    if (get_var("WORKAROUND_DEPS")) {
+    if (check_var("WORKAROUND_DEPS", '1')) {
         y2_logs_helper::workaround_dependency_issues;
     }
-    elsif (get_var("BREAK_DEPS")) {
+    elsif (check_var("BREAK_DEPS", '1')) {
         y2_logs_helper::break_dependency;
     }
```
Comment (coolgw) [inline lib/y2_installbase.pm]: If you change , then need cover all scenario, such as: zdup
lib/y2_installbase.pm:101:    if (get_var("WORKAROUND_DEPS")) {
tests/installation/change_desktop.pm:35:    if (check_screen('dependency-issue', 5) && get_var("WORKAROUND_DEPS")) {
tests/installation/zdup.pm:110:            if (get_var("WORKAROUND_DEPS")) {

File: lib/y2_installbase.pm [hunk 2]
```diff
@@ -98,10 +98,10 @@ sub deal_with_dependency_issues {
         send_key_until_needlematch 'packages-section-selected', 'tab';
         send_key 'ret';
     }
-    if (get_var("WORKAROUND_DEPS")) {
+    if (check_var("WORKAROUND_DEPS", '1')) {
         y2_logs_helper::workaround_dependency_issues;
     }
-    elsif (get_var("BREAK_DEPS")) {
+    elsif (check_var("BREAK_DEPS", '1')) {
         y2_logs_helper::break_dependency;
```
Comment (coolgw) [inline lib/y2_installbase.pm]: also for the BREAK_DEPS
 grep -risn BREAK_DEPS *
lib/y2_installbase.pm:104:    elsif (get_var("BREAK_DEPS")) {
tests/autoyast/installation.pm:128:    push(@needles, 'manual-intervention') if get_var("BREAK_DEPS");
tests/installation/change_desktop.pm:38:    if (check_screen('dependency-issue', 0) && get_var("BREAK_DEPS")) {

**Comment by hjluo** [comment]
@coolgw please review it again.

**Comment by lemon-suse** [comment]
LGTM.

**Pull request merged**